macros/SpecName.ejs

```diff
@@ -1138,4 +1142,4 @@ if ($2) {
 }
 
 
-%><%- result %>
+%><%- result %>

```

Comment:
This looks to me like a false positive. The only thing different is what line it appears on.